geom/geomUtils.cpp

```diff
@@ -45,19 +45,19 @@ using namespace utils;
 
 static int ss_default_color_ind = 0;
 static const char * ss_xgraph_colors[] =
-  {"black", "white", "red", "blue", "green", "violet", // 0,  ..., 5
-   "orange", "yellow", "pink", "cyan", "#A2B5CD",      // 6,  ..., 10
-   "#6C7B8B", "#FF00FF", "#00CDCD", "navy", "gold"     // 11, ..., 15
-  };
+{   "black", "white", "red",  "green", "cyan", "magenta",
+    "yellow", "pink", "teal", "aquamarine", "gold",
+    "brown", "blue",  "violet", "orange", "navy"
+};
 
 void setCurrentDefaultColor(int color){
   ss_default_color_ind = color;
 }
 
 std::string getCurrentDefaultColor() {
   ss_default_color_ind++;
-  if (ss_default_color_ind >= 15){
-    ss_default_color_ind -= 14; // start from 1
+  if (ss_default_color_ind >= 16){
```

Comment:
This could probably benefit from having ss_xgraph_colors and then using here .size(), so one does not forget the bound.

Reply:
I meant, std::vector<string>  ss_xgraph_colors